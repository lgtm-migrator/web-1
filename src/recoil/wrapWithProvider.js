import React from "react"
import { RecoilRoot } from "recoil"

// eslint-disable-next-line react/display-name,react/prop-types
export const wrapWithProvider = ({ element }) => (
  <RecoilRoot>{element}</RecoilRoot>
)
