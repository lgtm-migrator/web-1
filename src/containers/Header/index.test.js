import React from "react"
import renderer from "react-test-renderer"
import { RecoilRoot } from "recoil"
import { Header } from "."

describe("<Header /> container", () => {
  it("should render", () => {
    const tree = renderer
      .create(
        <RecoilRoot>
          <Header />
        </RecoilRoot>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
