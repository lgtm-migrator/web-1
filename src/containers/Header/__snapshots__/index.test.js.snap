// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> container should render 1`] = `
<header
  className="MuiBox-root MuiBox-root-42 MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
  itemScope={true}
  itemType="https://schema.org/WPHeader"
  role="banner"
>
  <div
    className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
  >
    <h1
      className="MuiTypography-root MuiTypography-body1"
    >
      <a
        className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
        href="/"
        onBlur={[Function]}
        onFocus={[Function]}
        target="_self"
      >
        
      </a>
    </h1>
  </div>
</header>
`;
